stl/inc/condition_variable

```diff
@@ -11,6 +11,7 @@
 #include <chrono>
 #include <memory>
 #include <mutex>
+#include <stop_token>

```

Comment:
Users including `<condition_variable>` in C++14/17 mode will be warned that `<stop_token>` requires C++20.

Reply:
To prevent this, wrap in a `#if _HAS_CXX20` block I would assume?
If so, should I directly place this with the other includes which may appear messy, or, should I place the 
```
#if _HAS_CXX20
#include <stop_token>
#endif // _HAS_CXX20
```
after all of the includes?

Reply:
> To prevent this, wrap in a `#if _HAS_CXX20` block I would assume?

Yep. Placing it after the rest of the includes seems reasonable, since there is no order dependency.